core/modules/widgets/slot.js

```diff
@@ -43,13 +43,24 @@ Compute the internal state of the widget
 SlotWidget.prototype.execute = function() {
 	var self = this;
 	this.slotName = this.getAttribute("$name");
-	// Find the parent transclusion
-	var transclusionWidget = this.parentWidget;
-	while(transclusionWidget && !(transclusionWidget instanceof TranscludeWidget)) {
-		transclusionWidget = transclusionWidget.parentWidget;
+	this.slotDepth = parseInt(this.getAttribute("$depth","1"),10) || 1;
```

Comment:
There is `$tw.utils.parsInt()` which returns 0 as fallback. IMO your `|| 1` at the end can be skipped, since the `.getAttribute()` already makes sure there will be a 1 as default.